sqladmin/models.py

```diff
@@ -588,6 +647,20 @@ def get_form_columns(self) -> List[Tuple[str, Column]]:
             default=self.get_model_attributes,
         )
 
+    def get_export_columns(self) -> List[Tuple[str, Column]]:
+        """Get list of columns to export."""
+
+        columns = getattr(self, "column_export_list", None)
+        excluded_columns = getattr(self, "column_export_exclude_list", None)
+        if not columns and not excluded_columns:
+            return self.get_list_columns()
+
+        return self._build_column_list(
+            include=columns,
+            exclude=excluded_columns,
+            default=lambda: [getattr(self.model, self.pk_column.name).prop],
```

Comment:
```suggestion
            default=lambda: self._list_columns,
```

Reply:
(I opted to not do it because it relies on execution order. But beneficially, it also more strongly couples the default behavior to copying list columns. Bit of a trade-off...)